### PR TITLE
feat(infra): Update warp configs in infra

### PR DESCRIPTION
**Pull request by Mo-Hussain**
### Description

- update the warp configs in infra to reflect expected behavior or change behavior

### Drive-by changes

- allow fork script to support passing warp route id

### Testing

Manual

### CODE REVIEW

File: typescript/infra/config/environments/mainnet3/warp/configGetters/getArbitrumNeutronTiaWarpConfig.ts [hunk 1]
```diff
@@ -19,18 +18,6 @@ export const getArbitrumNeutronTiaWarpConfig = async (
 
   const arbitrum: TokenRouterConfig = {
     ...routerConfig.arbitrum,
```
Comment (Mo-Hussain) [inline typescript/infra/config/environments/mainnet3/warp/configGetters/getArbitrumNeutronTiaWarpConfig.ts:20]: Set to default ISM

File: typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumInevmUSDCWarpConfig.ts [hunk 2]
```diff
@@ -5,31 +5,18 @@ import {
...
-    ['inevm'],
-    defaultMultisigConfigs,
-  ).inevm;
-
   const ethereum: TokenRouterConfig = {
     ...routerConfig.ethereum,
     type: TokenType.collateral,
     token: tokens.ethereum.USDC,
-    hook: '0xb87AC8EA4533AE017604E44470F7c1E550AC6F10', // aggregation of IGP and Merkle, arbitrary config not supported for now
-    interchainSecurityModule: ismConfig,
```
Comment (Mo-Hussain) [inline typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumInevmUSDCWarpConfig.ts:32]: Set to default ISM

File: typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumInevmUSDTWarpConfig.ts [hunk 3]
```diff
@@ -5,30 +5,18 @@ import {
...
-    ['inevm'],
-    defaultMultisigConfigs,
-  ).inevm;
-
   const ethereum: TokenRouterConfig = {
     ...routerConfig.ethereum,
     type: TokenType.collateral,
     token: tokens.ethereum.USDT,
-    hook: '0xb87AC8EA4533AE017604E44470F7c1E550AC6F10',
-    interchainSecurityModule: ismConfig,
```
Comment (Mo-Hussain) [inline typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumInevmUSDTWarpConfig.ts:31]: Set to default ISM

File: typescript/infra/config/environments/mainnet3/warp/configGetters/getMantapacificNeutronTiaWarpConfig.ts [hunk 4]
```diff
@@ -19,18 +18,6 @@ export const getMantapacificNeutronTiaWarpConfig = async (
 
   const mantapacific: TokenRouterConfig = {
     ...routerConfig.mantapacific,
-    interchainSecurityModule: {
```
Comment (Mo-Hussain) [inline typescript/infra/config/environments/mainnet3/warp/configGetters/getMantapacificNeutronTiaWarpConfig.ts:22]: Set to default ISM

File: typescript/infra/fork.sh [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ENVIRONMENT=$1
 MODULE=$2
 CHAIN=$3
+WARP_ROUTE_ID=$4
```
Comment (Mo-Hussain) [inline typescript/infra/fork.sh:5]: Support warp route ID

Comment (tkporter) [reply typescript/infra/fork.sh:5]: just wanna confirm that if you're running this for e.g. core, the cli arg parsing still works / is fine with just receiving `--warpRouteId`

**Review by tkporter — APPROVED** [review]

File: typescript/infra/fork.sh (hunk 5 above)
Comment (Mo-Hussain) [reply typescript/infra/fork.sh:7]: Addresse!

**Pull request merged**